core/trino-main/src/test/java/io/trino/execution/TestDropTableTask.java

```diff
@@ -63,6 +63,24 @@ public void testDropNotExistingTableIfExists()
         // no exception
     }
 
+    @Test
+    public void testDropNotExistingSchemaIfExists()
```

Comment:
"testDropNotExistingSchema..." sounds like dropping a schema. How about `testDropTableIfExistsWithoutExistingSchema` or something?

Reply:
Yes sounds good. will make the changes.